.github/CONTRIBUTING.md

```diff
@@ -3,6 +3,17 @@
 We'd love to accept your patches and contributions to this project. There are
 just a few small guidelines you need to follow.
 
+## Changes within the Proto Folder
+
+Changes within the `proto` folder are expected to be performed after
+arriving at mutual consensus within Google (although pull requests
+may be used to arrive at that consensus).
+
+Non-trivial changes to the schema should also include a suitable
```

Comment:
I don't understand why we're no longer asking people to generate the registry. What's keeping the catalog/schema/README consistent now?

Reply:
I did not change any part of the wording of this contributing section that was originally in the main README. I just moved the note that was authored when we only had protos.

We have CI keeping the files consistent. I expect any person to ensure the CI passes before sending a PR.

Reply:
This section had more details in the README than you've included here - in particular, it gave advance notice that they needed to regenerate the registry and said how to do so.

I think it's friendlier to give that notice up-front rather than have folks create a PR that then fails CI because they don't know they need to run other tools. It's fine just to say that regeneration is required and link to another doc though - it doesn't need to be in any depth.

Reply:
So the CI PR has details on how to run the generator which gives up-front notice. I've added more details in this PR too.

https://github.com/googleapis/google-cloudevents/pull/117/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R60